schemas/log/log.json

```diff
@@ -0,0 +1,40 @@
+{
+    "title": "Log",
+    "description": "A device log message",
+    "type": "object",
+    "properties": {
+      "appId": {
+          "type": "string",
+          "const": "LOG"
+      },
+      "ts": {
+          "type": "number",
+          "description": "Unix timestamp given in milliseconds when the data was sampled"
+      },
+      "seq": {
+          "type": "number",
+          "description": "Sequence number of the message. This is a counter, used for ordering message if `ts` not available/used"
+      },
+      "domain": {
```

Comment:
i realize "domain" is from [zephyr](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/services/logging/index.html#multi-domain-support), but would "source" ("src") be more clear to a user?

Reply:
Hmm. I have no informed opinion about the terminology a firmware dev would find most intuitive in this case.

Reply:
@plskeggs suggested this field. Maybe he has opinions?

Reply:
There is already a source as part of the log structure inside Zephyr, and it is a string.  On the device side, I was planning to include that as a prefix to the log string itself.  Source in this context is the module in NCS or Zephyr generating the log, or the user's application itself.  For example, "application", "nrf_cloud_pgps", "nrf_cloud_transport", etc.

Domain is a different animal.  It is a number representing which processor core in a multi-core SoC generated the log message.

Reply:
@plskeggs Should I include this as part of the description of domain?

Reply:
yes